sqlglot/expressions.py

```diff
@@ -12,7 +13,6 @@ def __init__(self, **args):
         self.args = args
         self.validate()
         self._parent = None
-        self.arg_key = None
```

Comment:
why is this removed?

Reply:
Ah, good catch. Accidental change. Btw, I don't see it being used anywhere except when we set it during parser. What's it for?

Reply:
it's so when you walk, when you yield, you know what the parent's key of the token was